iFixit/react-commerce
# Troubleshooting: Add styling to links

This makes links the blue color shown in the mockup.

## QA Notes
Try https://react-commerce-git-vulcan-add-link-style-ifixit.vercel.app/Vulcan/troubleshooting-test as a page which should include a link.

Closes https://github.com/iFixit/ifixit/issues/46334
